src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/enums/MailboxType.java

```diff
@@ -0,0 +1,21 @@
+package de.dhbw.karlsruhe.students.mailflow.core.domain.email.enums;
+
+public enum MailboxType {
+  COMMON("common"),
```

Comment:
I don't really seem to understand the `common` mailbox type... maybe a doc comment would help here to clarify the subject-matter (Fachlichkeit)?

Reply:
Good point! I added the javadoc and removed the Common type.
We also discussed that the isRead field of the Email class is not needed anymore as it will be defined by the mailbox. I removed it from the Email class